lib/streamlit/elements/plotly_chart.py

```diff
@@ -189,7 +189,11 @@ def plotly_chart(
         key: Key | None = None,
         on_select: Literal["ignore"],  # No default value here to make it work with mypy
         selection_mode: SelectionMode
-        | Iterable[SelectionMode] = ("points", "box", "lasso"),
+        | Iterable[SelectionMode] = (
+            "points",
+            "box",
+            "lasso",
+        ),
```

Comment:
unrelated nit: we should try to get a standardized formatter in our project to avoud this kind of unrelated formats (or is it because the file was not touched for a long time?)

Reply:
I assumed it's possible it hasn't been formatted, however, I assume with @LukasMasuch 's work this didn't. I thought the precommit black was doing the work, however I know there's a new Python formatter in town, ruff.

Reply:
I believe the problem here is that historically, there were two different black versions in use with CI/pre-commit/make pylint.. This got fixed very recently; see this comment here: https://github.com/streamlit/streamlit/pull/8614#discussion_r1591149795 

From validation standpoint, both are valid formatted code in our CI check. 

> I know there's a new Python formatter in town, ruff.

I think that is a worthy goal to eventually fully migrate formatting to ruff and thereby also finally introduce some proper linting for our python code.